curlable.js

```diff
@@ -9,24 +9,35 @@ const bodyParser = require('body-parser');
 const curlable = require('./lib/curlable.js');
 
 program
-  .version('0.0.1')
+  .version('0.0.6')
   .option('-c, --cmdline <cmdline>', 'The command to run.')
   .option('-p, --port <port>', 'Post to listen on.', parseInt)
   .option('-r, --route <route>', 'The route to serve the command on.')
+  .option('-q, --prompt <prompt>', 'The ready-to-process prompt, for multiline outputs.')
   .parse(process.argv);
 
 let cmdline = program.cmdline || 'bc -l'; // `bc -l` is a good default. -- D.K.
 let port = program.port || 8000;
 let route = program.route || '/';
+let options = {
+  prompt: program.prompt || null,
+};
 
 process.stderr.write('Making `' + cmdline + '` curlable.\n');
+if (options.prompt) {
+  process.stderr.write('Multiline output mode, prompt: `' + options.prompt + '`.\n');
+}
 
-const instance = new curlable.Curlable(cmdline, (msg) => { process.stderr.write(msg + '\n'); });
+const instance = new curlable.Curlable(cmdline, options, (s) => { process.stderr.write(s); });
 
 curlable.readStreamByLines(process.stdin, (line) => {
-  instance.runQuery(line, (result) => {
-    process.stdout.write(result + '\n');
-  });
+  if (instance.ready) {
+    instance.runQuery(line, (result) => {
+      process.stdout.write(result);
```

Comment:
What if result doesn't contain `\n`, is that expected?

Reply:
It could be multline, easier to append each line with an `'\n` at the end.